source/content/guides/composer-convert.md

```diff
@@ -166,119 +157,130 @@ What makes your site code unique is your selection of contributed modules and th
 
 #### Modules and Themes
 
-THE GOAL: To have composer manage all your contrib modules, contrib themes, core upgrades and libraries. The only thing that should be checked in is custom code, custom themes and custom modules that are specific to your site and your site alone.
+The goal of this process is to have Composer manage all the site's contrib modules, contrib themes, core upgrades, and libraries (we'll call this _contributed code_). The only things that should be migrated from the existing site are custom code, custom themes, and custom modules that are specific to the existing site.
 
-WHY?: So that upgrading your Drupal install is a matter of `cd`-ing to the directory and running `composer upgrade`
+The steps here ensure that any modules and themes from [drupal.org](drupal.org) are in the `composer.json` `require` list.
 
-To serve this end: we need to make sure all of your modules and themes that were downloaded from drupal.org are in the `composer.json` "require" list.
+Once Composer is aware of all the contributed code, you'll be able to run `composer upgrade` from within the directory to have Composer upgrade all the contributed code automatically.
 
-A Composer-managed site should be able to include all custom code via Composer. Begin by reviewing your existing site's code. Check for contributed modules in `/modules`, `/modules/contrib`, `/sites/all/modules`, and `/sites/all/modules/contrib`.
+Begin by reviewing the existing site's code. Check for contributed modules in `/modules`, `/modules/contrib`, `/sites/all/modules`, and `/sites/all/modules/contrib`.
 
-1. When reviewing the site, take stock of exactly what versions of modules and themes you depend on. One way to do this is to use a command like the following from within a contributed modules folder (e.g. `/modules`, `/themes`, `/themes/contrib`, `/sites/all/themes`, `/sites/all/themes/contrib`, etc.).
+1. When reviewing the site, take stock of exactly what versions of modules and themes you depend on. One way to do this is to change to run the `pm:projectinfo` Drush command from within a contributed modules folder (e.g. `/modules`, `/themes`, `/themes/contrib`, `/sites/all/themes`, `/sites/all/themes/contrib`, etc.).
```

Comment:
Drush `pm:projectinfo` does not care what your cwd is. This command reads module information from the database, so it won't work on the local site unless the user has also set up a local database. This is why the recommended command below uses `terminus drush` & runs against the dev environment.